src/main/handlers/media/transcriptionEngines/dummyTranscribeFunction.ts

```diff
@@ -0,0 +1,45 @@
+import path from 'path';
+import fs from 'fs';
+import { MapCallback, PartialWord } from 'sharedTypes';
+import { TranscriptionFunction } from '../transcribeTypes';
+
+/**
+ * Replace the start_time attribute with startTime (can be generalised further but shouldn't
+ * need this once python outputs camelcase anyway)
+ * @param word snake cased partial word
+ * @returns camel cased partial word
+ *
+ */
+const camelCase: MapCallback<SnakeCaseWord, PartialWord> = (word) => ({
+  word: word.word,
+  duration: word.duration,
+  startTime: word.start_time,
+});
+
+interface SnakeCaseWord {
```

Comment:
Don't copy these interfaces and utils between files, make them shared - I don't even think they're used here anyway

Reply:
Yep, chucked it in a shared file. The deepspeech stuff is gonna get deleted soon, so not gonna stress too much about it.